### DISCUSSION TITLE
Help implementing multi touch gestures (resize & rotate)

### DISCUSSION THREAD

**Discussion by jaypeecarreon** [fabricjs/fabric.js#8385] [Q&A]
Hi was wondering how to implement multi touch gestures when a user is viewing a canvas on mobile. Giving the user the ability to resize and rotate an object. Trying to implement this in my react app.

**Comment by ShaMan123** [comment]
https://github.com/fabricjs/fabric.js/blob/master/src/mixins/canvas_gestures.mixin.ts

**Reply by ShaMan123** [reply]
you need to build fabric with gestures I believe

**Reply by ShaMan123** [reply]
use v5
```bash
 node build.js modules=ALL exclude=accessors
```

**Reply by ShaMan123** [reply]
v6 with be handled differently
probably an import

**Reply by jaypeecarreon** [reply]
Any docs/tutorials you recommend that have implementation details for this?

**Reply by ShaMan123** [reply]
Not that I know of
Did you check the website?
I would simply read it through

**Reply by melchiar** [reply]
@jaypeecarreon gesture resizing/rotating is on by default when you run fabric with the gesture module included (no setup required)
